Framework/CurveFitting/inc/MantidCurveFitting/RalNlls/TrustRegion.h

```diff
@@ -19,19 +19,19 @@ void mult_Jt(const DoubleFortranMatrix &J, const DoubleFortranVector &x,
 double evaluate_model(const DoubleFortranVector &f,
                       const DoubleFortranMatrix &J,
                       const DoubleFortranMatrix &hf,
-                      const DoubleFortranVector &d, const nlls_options options,
+                      const DoubleFortranVector &d, const nlls_options &options,
                       evaluate_model_work &w);
 double calculate_rho(double normf, double normfnew, double md,
                      const nlls_options &options);
 void update_trust_region_radius(double &rho, const nlls_options &options,
                                 nlls_inform &inform, NLLS_workspace &w);
-void rank_one_update(DoubleFortranMatrix &hf, NLLS_workspace w);
+void rank_one_update(DoubleFortranMatrix &hf, NLLS_workspace &w);
```

Comment:
So does this fix a bug then? It seems to be used in a few places via `TrustRegionMinimizer`, so I guess this should be tested and in the release notes?